### PR TITLE
Add Rocky Linux 9 Keys and Refresh Rocky 8 Keys

### PR DESCRIPTION
**Pull request by nazunalika**
Hello. This PR does the following:

* Add the official Rocky Linux 9 GPG key for the upcoming July 14th release
* Refresh the Rocky Linux 8 key to have consistent naming with 9 and future versions

### CODE REVIEW

**Comment by nazunalika** [comment]
Hello. You are right, I did not consider the implication of renaming the key and folks may be using old mock configs. I pushed a new commit with a symlink, so hopefully that works.

File: SOURCES.md [hunk 1]
```diff
@@ -156,7 +156,9 @@ https://git.remirepo.net/cgit/rpms/remi-release.git/plain/RPM-GPG-KEY-remi2022
 
 ## Rocky Linux
 
-https://dl.rockylinux.org/pub/rocky/8/BaseOS/x86_64/os/RPM-GPG-KEY-rockyofficial
+https://dl.rockylinux.org/pub/rocky/8/BaseOS/x86_64/os/RPM-GPG-KEY-Rocky-8
+
+https://dl.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/RPM-GPG-KEY-Rocky-9
```
Comment (xsuchy) [inline SOURCES.md:161]: This URL returns 404. The correct address seems to be https://dl.rockylinux.org/pub/rocky/RPM-GPG-KEY-Rocky-9
Ditto for the URL of RPM-GPG-KEY-Rocky-8
And can you add the Testing and infra keys as well?

Comment (nazunalika) [reply SOURCES.md]: Good catch! Thank you for that. I hope I've addressed it now. Let me know if I missed anything else!

**Comment by xsuchy** [comment]
Thank you.

**Pull request merged**